search-endpoint.md

```diff
@@ -0,0 +1,397 @@
+# Discovery Search Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Search Endpoint
+**/api/discover/search**   
```

Comment:
I don't like `searches`. I would either stick with the verb `search` or go with a noun like `objects` (so that would make `/api/discover/objects` which is OK I think).